cookiecutter/log.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+
+import logging
+import sys
+
+LOG_LEVELS = {
+    'DEBUG': logging.DEBUG,
+    'INFO': logging.INFO,
+    'WARNING': logging.WARNING,
+    'ERROR': logging.ERROR,
+    'CRITICAL': logging.CRITICAL,
+}
```

Comment:
If I understand correctly ``getLevelName`` returns a string for a int. I'd like to go the other way so we don't need to pass around ints.